test/Groups/Permutations.jl

```diff
@@ -427,3 +427,39 @@ end
   H,iso = smaller_degree_permutation_representation(G)
   @test degree(H)<degree(G)
 end
+
+@testset "fixed_points tests" begin
+  # Setup permutation group
+  g = symmetric_group(4)
+  s = sylow_subgroup(g, 3)[1]        # a Sylow 3-subgroup (order 3)
+  e = one(g)                         # identity element
+  y = gen(g, 1)                      # an example element of g
+  x = gen(s, 1)                      # generator of Sylow subgroup
+
+  # Tests for fixed_points on individual elements
+  @test fixed_points(x) == [4]       # Sylow generator fixes only 4
+
+  @test fixed_points(e) == collect(1:degree(g))  # Identity fixes all
+
+  @test fixed_points(y) == [i for i in 1:degree(g) if y(i) == i]  # Check example element
+
+  z = g([2, 3, 1, 4])                # permutation (1 2 3)(4)
+  @test fixed_points(z) == [4]
+end
+
+@testset "number_of_fixed_points tests" begin
+  # Setup permutation group
+  g = symmetric_group(4)
+  s = sylow_subgroup(g, 3)[1]         # a Sylow 3-subgroup (order 3)
+  e = one(g)                          # identity element
+  y = gen(g, 1)
+  x = gen(s, 1)                       # generator of Sylow subgroup
+
+  # Tests for number_of_fixed_points on individual elements
+  @test number_of_fixed_points(x) == 1          # Sylow generator fixes only 4
+  @test number_of_fixed_points(e) == 4          # Identity fixes all points
+  @test number_of_fixed_points(y) == 0          # generator (1 2 3 4) fixes nothing
+
+  z = g([2, 3, 1, 4])                            # permutation (1 2 3)(4)
+  @test number_of_fixed_points(z) == 0
```

Comment:
```suggestion
  @test number_of_fixed_points(z) == 1
```
as you write in line 463, there is a 1-cycle containing 4, and thus 4 is a fixed point.